client/src/components/termsPrompt/index.js

```diff
@@ -26,13 +25,13 @@ function storageSet(key, value) {
   try {
     window.localStorage.setItem(key, value);
   } catch {
-    return;
+    // continue
```

Comment:
this seems error prone, if we were to ever add code after the try/catch.   Continue and return have different semantics.

Reply:
Looks like there were two steps to get to this result.  Running auto fix on the file led to the removal of `return` as it was unnecessary but the introduction of an empty block statement.   `no-empty` has the condition:  

> This rule ignores block statements which contain a comment (for example, in an empty catch or finally block of a try statement to indicate that execution should continue regardless of errors).

So the comment was me getting rid of the new eslint error, didn't consider the semantic meaning behind continue when making the comment.  Would you suggest a new comment such as: 

> // continue regardless of error

Reply:
thinking about this more, I don't have a great suggestion.  And the code is simple enough to ignore this.  So lets leave as is, and I"ll resolve this comment.